numpy/lib/polynomial.py

```diff
@@ -272,7 +272,7 @@ def polyint(p, m=1, k=None):
     Parameters
     ----------
     p : array_like or poly1d
-        Polynomial to differentiate.
+        Polynomial to integrate.
```

Comment:
What is this change doing in the PR??

Reply:
reverted rebase cruft

Reply:
I had to rebase to remove the two stray commits that didn't belong